pyproject.toml

```
@@ -61,7 +61,7 @@ sqlalchemy = "^1.3.19"
 structlog = "^21.2.0"
 tzlocal = "^4.2.0"
 uvicorn = {extras = ["standard"], version = "^0.17.6"}
-werkzeug = "^2.1"
+werkzeug = "<=2.1.3"
```

```suggestion
werkzeug = ">=2.1,<=2.1.3"
```

`poetry.lock` will have to be re-generated if this change is applied.